extensions/EXT_disjoint_timer_query/extension.xml

```diff
@@ -254,6 +253,69 @@
           // least significant 32 bits.
           adjustObjectLODBasedOnDrawTime(timeEnd - timeStart);
         }
+
+        //----------------------------------------------------------------------
+
+        // Example (3) -- check the number of timestamp bits to determine how to best
+        // measure elapsed time.
+        var ext = gl.getExtension('EXT_disjoint_timer_query');
+        var timeElapsedQuery;
+        var startQuery;
+        var endQuery;
+
+        var useTimestamps = false;
+
+        if (ext.getQueryEXT(ext.TIMESTAMP_EXT, ext.QUERY_COUNTER_BITS_EXT) > 0) {
+          useTimestamps = true;
+        }
+
```

Comment:
Clear gl.getParameter(ext.GPU_DISJOINT_EXT) right before starting queries.
It's technically cleared in the previous example, but it's good for this example to be self-contained.